dvid_point_cloud/client.py

```diff
@@ -76,6 +78,21 @@ def get_sparse_vol_stats(self, uuid: str, instance: str, label_id: int) -> Spars
             max_voxel=tuple(data["maxvoxel"])
         )
     
+    def get_supervoxels(self, uuid: str, instance: str, body_id: int) -> bytes:
```

Comment:
Probably better typing is:

from numpy.typing import NDArray
def get_supervoxels(self, uuid: str, instance: str, body_id: int) -> NDArray[np.int64]: